frontend/challedu-app/src/UserProvider.js

```diff
@@ -7,16 +7,16 @@ export default class UserProvider {
   static async getUser(id, access_token) {
     return await APIUtils.get(`users/${id}`, access_token);
   }
-  static async createUser(userData) {
-    return await APIUtils.post("users/", userData);
+  static async createUser(userData, access_token) {
+    return await APIUtils.post("users/", userData, access_token);
   }
-  static async updateUser(userData) {
-    return await APIUtils.put("users/", userData);
+  static async updateUser(id, userData, access_token) {
+    return await APIUtils.put(`users/${id}`, userData, access_token);
   }
-  static async deleteUser(id) {
-    return await APIUtils.delete(`users/${id}`);
+  static async deleteUser(id, access_token) {
+    return await APIUtils.delete(`users/${id}`, access_token);
   }
-  static async loginUser(userData) {
-    return await APIUtils.post(`auth/login/`, userData);
+  static async loginUser(userData, access_token) {
```

Comment:
Neither this one.

Reply:
@zannis . I pushed it, please check it out and merge it if it 's OK.